DocumentedExamples/Cross-contour_transport.ipynb

```diff
@@ -0,0 +1,1480 @@
+       "                            <strong>Nanny: </strong> tcp://127.0.0.1:39435\n",
+       "                        </td>\n",
+       "                        <td style=\"text-align: left;\"></td>\n",
+       "                    </tr>\n",
+       "                    <tr>\n",
+       "                        <td colspan=\"2\" style=\"text-align: left;\">\n",
+       "                            <strong>Local directory: </strong> /local/g40/cy8964/tmp/dask-worker-space/worker-3jafsg6k\n",
+       "                        </td>\n",
+       "                    </tr>\n",
+       "\n",
+       "                    \n",
+       "\n",
+       "                    \n",
+       "\n",
+       "                </table>\n",
+       "            </details>\n",
+       "            </div>\n",
+       "        </div>\n",
+       "        \n",
+       "\n",
+       "    </details>\n",
+       "</div>\n",
+       "\n",
+       "        </details>\n",
+       "    </div>\n",
+       "</div>\n",
+       "            </details>\n",
+       "        \n",
+       "\n",
+       "    </div>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "<Client: 'tcp://127.0.0.1:42035' processes=4 threads=16, memory=44.92 GiB>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client = Client(n_workers=4)\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0fc07fcc-6e1d-40c0-be5b-c887266a20c7",
+   "metadata": {},
+   "source": [
+    "#### Choose database"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "142873eb-d834-40c2-aa5c-7b8f750f83d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session = cc.database.create_session('/g/data/ik11/databases/cosima_master.db')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de04e5ed-ee85-48ff-9628-abd3e6d8ea7b",
+   "metadata": {},
+   "source": [
+    "#### Choose experiment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "eed34fea-7cdd-4439-9668-3f543b9a4766",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "expt = '01deg_jra55v13_ryf9091'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "63761ef6-84b9-4c3b-9f99-9f90cb878e49",
+   "metadata": {},
+   "source": [
+    "#### Choose a latitude range so the contour fits in the range, but there is not too much extra space. Extra space slows down the computation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "900d12f3-548f-4854-8686-5dc9f3d037c7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lat_range = slice(-64.99,-47)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ed43746-1772-4db8-be15-19accc9907ab",
+   "metadata": {},
+   "source": [
+    "#### We must make sure that this latitude range is so that the t-cells are always south and west of the u-cells.\n",
```

Comment:
what do you mean by "We must make sure that this latitude range is so that the t-cells are always south and west of the u-cells."? It's not clear to me.